src/components/Good.tsx

```diff
@@ -0,0 +1,12 @@
+type GoodProps = {
+  goodName: string;
+  key: number;
+};
+
+export const Good: React.FC<GoodProps> = ({ goodName, key }) => {
+  return (
+    <li data-cy="Good" key={key}>
```

Comment:
The `key` prop should not be applied here within the component. It should be used in the parent component where the list of `Good` components is rendered.